trieste/objectives/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 The Trieste Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This package contains examples of popular objective functions used in (Bayesian) optimization.
+"""
+
+from . import multi_objectives, observer, single_objectives
```

Comment:
`observer.py` should probably be a top level trieste module, like space.py, rather than part of the objectives package.

Also wondering whether it might make sense to `import * from single_objectives` here, so you can just write `from trieste.objectives import branin` (but still need `from trieste.objectives.multi_objectives import VLMOP2` for the multi objective functions).

Reply:
Also make sure you note the renaming in the git commit message as this will need to go into the release notes for the next release.

Reply:
here `observer.py` is util function for making an observer from the objective function, so I think it should be here, although we can name the file differently...

I think I would agree with the second suggestion, but not a strong preference